java/runtime/src/main/java/io/ray/runtime/exception/BackPressureException.java

```diff
@@ -0,0 +1,22 @@
+package io.ray.runtime.exception;
+
+import io.ray.api.id.ActorId;
+
+/**
+ * Indicates that the back pressure occurs when submitting an actor task.
+ *
+ * <p>This exception could happen either because the caller calls the callee too frequently.
+ */
+public class BackPressureException extends RayException {
+
```

Comment:
Tthe name of `BackPressureException` is more about the implementation details. To expose to the userend, it's better to be some thing like `PendingTasksFullException` or `TooManyPendingTaskException`.

Reply:
make sense. @edoakes what about python side? we'd better to keep python and java consistent.

Reply:
I'm not sure why `BackPressureException` is exposing implementation details, I like it because it is very clear what this exception is meant to be used for. I would also be ok with `MaxPendingCallsExceeded`.